toqito/matrices/hadamard.py

```diff
@@ -4,10 +4,10 @@
 
 
 def hadamard(n_param: int = 1) -> np.ndarray:
-    r"""Produce a :code:`2^{n_param}` dimensional Hadamard matrix :cite:`WikiHadamard`.
+    r"""Produce a :code:`2^{n_param}` dimensional Hadamard matrix :footcite:`WikiHadamard`.
```

Comment:
not needed as this already exists in the main branch. 

https://github.com/vprusso/toqito/blob/40d2326b328b2b855efe42d1f811912b481fddf0/toqito/matrices/hadamard.py#L7